### ISSUE TITLE
Error message when trying to download images

### ISSUE THREAD

**Issue by ninryu** [furyutei/twOpenOriginalImage#31]
Today I was suddenly unable to download images with the addon and got the error message: "Download failure:
https://pbs.twimg.com/media/F3hZ2csa0AAtMoi?format=jpg&name=orig".

**Comment by skrubbybubby** [comment]
Yes, same here. This includes both clicking download as well as creating a zip. The noted zip creates the image with a filesize of 0 bytes.
Right clicking and save as, as well as right clicking the image to a new tab work, but don't rename the image so I must do that manually.

**Comment by skrubbybubby** [comment]
Forgot to mention this works on my pc but not my laptop, but regardless, I installed the script version today instead of the extension version and it's working as intended

**Comment by Fuewburvpoa** [comment]
This is what i've noticed, it would stop working but after some time (day for example) it will work.

Next thing that i've notices is that when it does not work, URL that is generated by addon has `format=webp` as one of the parameters.

[Example tweet](https://twitter.com/sylpi_hp1/status/1695688927779471862) will generate this [https://pbs.twimg.com/media/F4hMEYUakAAQmBJ?format=webp&name=orig](https://pbs.twimg.com/media/F4hMEYUakAAQmBJ?format=webp&name=orig) as you can see `format=webp` if you change it to `format=jpg` then link will work
